### PR TITLE
Scraper/palmbeachstate

### PR DESCRIPTION
**Pull request by HarrisonGregg**

### CODE REVIEW

File: osp_scraper/spiders/palmbeachstate.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+        def parse_people_results(response):
+            for row in response.css("#ContentPlaceHolder1_gvResults tr"):
+                relative_url = row.css("a::attr(href)").extract_first()
+                if relative_url:
+                    url = response.urljoin(relative_url)
+                    name = row.css("a::text").extract_first()
+
+                    yield scrapy.Request(
+                        url,
```
Comment (jodizzle) [inline osp_scraper/spiders/palmbeachstate.py]: Minor: Trailing whitespace.

File: osp_scraper/spiders/palmbeachstate.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                            "depth": 1,
+                            "hops_from_seed": 1,
+                            "source_url": response.url,
+                            "source_anchor": name
+                        },
+                        callback=self.parse_for_files
+                    )
+
+        yield scrapy.FormRequest(
+            start_url,
```
Comment (jodizzle) [inline osp_scraper/spiders/palmbeachstate.py]: Minor: Trailing whitespace.

File: osp_scraper/spiders/palmbeachstate.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+from ..spiders.CustomSpider import CustomSpider
+
+class PalmBeachStateSpider(CustomSpider):
+    name = "palmbeachstate"
+
+    def start_requests(self):
+        start_url = "http://www.palmbeachstate.edu/pf/"
+        start_formdata = {
+            "__VIEWSTATE":"/wEPDwUKLTgyOTAxMzE2MA9kFgJmD2QWBAIJDw8WAh4EVGV4dAUNUGVvcGxlIEZpbmRlcmRkAgsPZBYCAgEPZBYGAgEPZBYEAgEPZBYCAgcPEA8WAh4LXyFEYXRhQm91bmRnZBAVBgAKTGFrZSBXb3J0aBJQYWxtIEJlYWNoIEdhcmRlbnMLQmVsbGUgR2xhZGUSTG94YWhhdGNoZWUgR3JvdmVzCkJvY2EgUmF0b24VBgACTFcDUEJHAkJHAkxHAkJSFCsDBmdnZ2dnZ2RkAgMPZBYCAgMPEA8WAh8BZ2QQFQYACkxha2UgV29ydGgSUGFsbSBCZWFjaCBHYXJkZW5zC0JlbGxlIEdsYWRlEkxveGFoYXRjaGVlIEdyb3ZlcwpCb2NhIFJhdG9uFQYACkxha2UgV29ydGgSUGFsbSBCZWFjaCBHYXJkZW5zC0JlbGxlIEdsYWRlEkxveGFoYXRjaGVlIEdyb3ZlcwpCb2NhIFJhdG9uFCsDBmdnZ2dnZ2RkAgMPDxYCHgdWaXNpYmxlaGQWAgIBDzwrABECARAWABYAFgAMFCsAAGQCBQ8PFgIfAmhkFgICAQ88KwARAgEQFgAWABYADBQrAABkGAIFJ2N0bDAwJENvbnRlbnRQbGFjZUhvbGRlcjEkZ3ZEZXB0UmVzdWx0cw9nZAUjY3RsMDAkQ29udGVudFBsYWNlSG9sZGVyMSRndlJlc3VsdHMPZ2QmuzHr7rNX0pVadDv8SQYNMJ2pLdWGyg4Fl3TmSqjpxg==",
+            "__SCROLLPOSITIONX":"0",
```
Comment (jodizzle) [inline osp_scraper/spiders/palmbeachstate.py]: Can you refactor to use `from_response` instead?  Or alternatively just extract the parameters from the HTML if you want to keep your current structure.

**Comment by HarrisonGregg** [comment]
@jodizzle Refactored to use `from_response`.  Looks a lot cleaner now and should be fully functional.

**Comment by jodizzle** [comment]
Looks good.  OK to merge.

**Pull request merged**